src/utils.c

```diff
@@ -1564,8 +1565,10 @@ parse_index_info(char *ns, char *index_str, index_param *index)
 				err("Invalid index type %s", arg);
 				goto cleanup2;
 			}
-		} else if (strcmp(para, "path") == 0 || strcmp(para, "context") == 0) {
+		} else if (strcmp(para, "bin") == 0) {
 			path = arg;
+		} else if (strcmp(para, "context") == 0 && strcasecmp(arg, "NULL") != 0) {
```

Comment:
question for my understanding. What necessitates the ` && strcasecmp(arg, "NULL") != 0)`?

Reply:
all the records for `sindex-list` command have a `context` parameter but only those with a not null value should be skipped.